tools/ci_build/build.py

```diff
@@ -1942,11 +1780,6 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
             if not args.disable_ml_ops and not args.use_tensorrt:
                 run_subprocess([sys.executable, "onnxruntime_test_python_mlops.py"], cwd=cwd, dll_path=dll_path)
 
-            # The following test has multiple failures on Windows
-            if args.enable_training and args.use_cuda and not is_windows():
-                # run basic frontend tests
-                run_training_python_frontend_tests(cwd=cwd)
```

Comment:
If these are going to be reenabled in the future, would it make sense to add a TODO or log with an explanation?

```python
if args.enable_training and args.use_cuda and not is_windows():
    # TODO: Add tests when ....
   log.info("...")
```

Reply:
These are not going to be re-enabled in the future. These tests are part of the older `ORTTrainer` code. This has been deprecated and removing the unit tests was part of the plan for a while.
There was only one test here that needs to be re-enabled, and I have moved it to https://github.com/microsoft/onnxruntime/pull/14055/files#diff-10e9937810323c48f9c951ef10a99ba7f16829893232d59584242499eccbc76fR189-R191